delivery.yaml

```diff
@@ -3,9 +3,8 @@ build_steps:
   cmd: |
     apt-get update
     apt-get install -q -y --no-install-recommends \
-      git \
-      python3.5 \
-      python3.5-dev \
+      python3.6 \
```

Comment:
Will it work with `python3` and `python3-dev` instead?

Reply:
Reverted to python3.5